File: src/sdk-reference/js/6/document/deleteByQuery/snippets/delete-by-query.js
try {
  const deleted = await kuzzle.document.deleteByQuery(
    'nyc-open-data',
    'yellow-taxi',
    {
      query: {
        match: {capacity: 7}
      }
    }
  );

  console.log('Success');
} catch (error) {
  console.error(error.message);
}

File: src/sdk/java/1/controllers/document/delete-by-query/snippets/delete-by-query.java
try {
    StringVector deleted = kuzzle.getDocument().deleteByQuery(
      "nyc-open-data",
      "yellow-taxi",
      "{\"query\": {\"term\": {\"capacity\": 7}}}"
    );

    System.out.println(String.format("Successfully deleted %d documents", deleted.size()));
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}
